_posts/2019/2019-09-18-you.md

```diff
@@ -0,0 +1,111 @@
+Vilkas is known for its high rate of customer satisfaction.
+Tatu emphasized that the key to keep customers satisfied is to support them in whatever they are doing.
+
+### It’s people-BINGO-time!
+
+To raise conversation beyond the familiar office doors, an ice breaker was organized.
+This year it was bingo.
+We each received a table with certain characteristics.
+The goal was to find people who were not working on your office floor and who match those characteristics, so that eventually you’d get a Bingo by putting their names in the table.
+It was quite fun and a good opportunity to get to know new people from the company.
+So at the end of the day, even if you know nothing else about that person, you’ll know if he or she owns a Madonna CD.
+
+### Know your customer
+
+After a short break we had a panel discussion with four merchants on the agenda.
+This was probably our favorite part because we got first-hand feedback in an open discussion from merchants directly using our software on a daily basis.
+They addressed what they already like about the current software but also which changes and features they would like to see in the future.
+
+{% twitter https://twitter.com/epagesdevs/status/1172078677452120064 %}
+
+It was great to see what we already achieved in terms of our online shop software but also to recognize what still needs to be done in order to meet the merchants needs.
+Thanks again to Kati Decker from superstreusel.de, Imme Hansemann from hansemann.de, Michael Mommsen from nutringo.de and Heinrich Dumrath from chronobalance.de for your time and opinion.
+
+### Work it
+
+After a little lunch break, we continued with several workshops for which we could sign up for during the last weeks.
+They all covered different topics, such as usability testing, customer micro-moments and journeys, and a perspective change game.
+
+{% image_custom image="/assets/img/pages/blog/images/blog-you19-2.jpg" width="50" lightbox %}
+
+After the end of the workshops, the different groups made their way back to the presentation room where each group presented what they had worked on for the past two hours.
+This way everyone got an impression of what the other workshops were about.
+
+### Keep calm and shoot straight
+
+Afterwards, we were ready for some action.
+This year’s team event was archery!
+Fortunately, we had the weather on our side.
+When we arrived at the location everything was already set up.
+All ePagees were divided into 20 teams and we got an introduction on how to use the bows and arrows correctly.
+Each player had eight test shots to get a first feeling and then a little competition started.
+Overall it was a very cool experience to try out this uncommon sport.
+
+{% image_custom image="/assets/img/pages/blog/images/blog-you19-1.jpg" width="50" lightbox %}
+
+The sport part was followed by the delicious part.
```

Comment:
Maybe better: "After sport comes the pleasure." Or "After work comes the please."